packages/gatsby-graphiql-explorer/src/app/snippets.js

```diff
@@ -69,4 +69,34 @@ export default ComponentName
 `,
 }
 
-export default [pageQuery, staticHook, staticQuery]
+const createPages = {
+  name: `createPages`,
+  language: `JavaScript`,
+  codeMirrorMode: `javascript`,
+  options: [],
+  generate: arg => `const path = require(\`path\`)
+
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage } = actions
+
+  const result = await graphql(\`
+${getQuery(arg, 4)}
+  \`)
+
+  result.data.${
+    arg.operationDataList[0].operationDefinition.selectionSet.selections[0].name
+      .value
+  }.edges.forEach(({ node }) => {
```

Comment:
We don't really teach people the `edges { node }` pattern anymore and let them use `nodes` directly

Reply:
Updated